Tools/CISupport/ews-build/master.cfg

```diff
@@ -28,6 +28,18 @@ c['www']['ui_default_config'] = {
     'Builders.buildFetchLimit': 1000,
     'Workers.showWorkerBuilders': True,
 }
+c['www']['change_hook_dialects'] = dict(
+    github=dict(
+        secret=os.getenv('GITHUB_HOOK_SECRET', None),
```

Comment:
maybe let's use passwords.json for this and try to move away from environment variables.